scripts/st-ctl.sh

```diff
@@ -458,20 +462,41 @@ if [ ! -e "/keep" ] || [ ! -e "/mnt/data/keep/" ]; then
   rm /keep
   ln -s /mnt/data/keep/ /keep
 fi
-
 cd /mnt/data/
+
 EOF
 
 # inside concord container
 docker exec -it ${concord_container_name} bash -c "apt update && apt install nano -y"  >/dev/null 2>&1 || true
-echo "Done Installing tools, please log in and out"
+
+. ~/.profile
+
+# install minio command line tool 'mc' in RO replica
+if [[ "$(my_id)" == *"id_ro"* ]]; then
+  rm -rf ${minio_command_line_tool_path}*
+  echo "Installing minio mc under /mnt/data/"
+  wget https://dl.min.io/client/mc/release/linux-amd64/mc
+  chmod +x ${minio_command_line_tool_path}
+  ip=$(cat /config/concord/config-local/deployment.config | grep -m1 s3-url | cut -d ":" -f 2)
+  port=$(cat /config/concord/config-local/deployment.config | grep -m1 s3-url | cut -d ":" -f 3)
+  secret_key=$(cat /config/concord/config-local/deployment.config | grep -m1 s3-secret-key | cut -d ' ' -f 6)
+  access_key=$(cat /config/concord/config-local/deployment.config | grep -m1 s3-access-key| cut -d ' ' -f 6)
+  ip=$(echo ${ip} | xargs)
+  port=$(echo ${port} | xargs)
+  secret_key=$(echo ${secret_key} | xargs)
+  access_key=$(echo ${access_key} | xargs)
+  echo "Adding alias: ./mc alias set min http://${ip}:${port} ${access_key} ${secret_key}"
+  ${minio_command_line_tool_path} alias set ${minio_alias_name}/ http://${ip}:${port} ${access_key} ${secret_key}
 fi
 
+echo "Done Installing tools, please log in and out"
```

Comment:
Maybe the second part of the message can be rewritten as `please relog`, since the user is already logged in.